src/SignalR/perf/benchmarkapps/Crankier/Server/Startup.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.SignalR.Crankier.Server
+{
+    public class Startup
+    {
+        private readonly IConfiguration _config;
+        public Startup(IConfiguration configuration)
+        {
+            _config = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var signalrBuilder = services.AddSignalR();
```

Comment:
nit: I would keep AddMessagePack as it doesn't affect anything and adds support for clients wanting to test with MessagePack

Reply:
Agreed. I know I asked for it to be removed earlier. Sorry for the confusion!

Reply:
done